src/test/java/uk/gov/hmcts/pdda/business/services/formatting/MergeListUtilsTest.java

```diff
@@ -0,0 +1,101 @@
+ * </p>
+ * <p>
+ * Copyright: Copyright (c) 2024
+ * </p>
+ * <p>
+ * Company: CGI
+ * </p>
+ * 
+ * @author Luke Gittins
+ */
+@ExtendWith(EasyMockExtension.class)
+class MergeListUtilsTest {
+
+    private static final String EQUAL = "Result is Equal";
+    private static final String FALSE = "Result is False";
+
+    private static final String[] DAILYLISTROOTNODES = {"DailyList/CourtLists"};
+    private static final String[] FIRMLISTROOTNODES = {"FirmList/CourtLists", "FirmList/ReserveList"};
+    private static final String[] WARNEDLISTROOTNODES = {"WarnedList/CourtLists/CourtList"};
+    
+    private static final String INVALID = "Invalid";
+
+    @BeforeAll
+    public static void setUp() throws Exception {
+        // Do nothing
+    }
+
+    @AfterAll
+    public static void tearDown() throws Exception {
+        // Do nothing
+    }
+
+    @Test
+    void testGetListRootNodesDailyList() throws XPathExpressionException, IOException {
+        DailyListXmlMergeUtils dailyListXmlMergeUtils = new DailyListXmlMergeUtils();
+        String[] result = MergeListUtils.getListRootNodes(dailyListXmlMergeUtils.getClass());
+        assertEquals(DAILYLISTROOTNODES[0], result[0], EQUAL);
+    }
+    
+    @Test
+    void testGetListRootNodesFirmList() throws XPathExpressionException, IOException {
+        FirmListXmlMergeUtils firmListXmlMergeUtils = new FirmListXmlMergeUtils();
+        String[] result = MergeListUtils.getListRootNodes(firmListXmlMergeUtils.getClass());
+        assertEquals(FIRMLISTROOTNODES[0], result[0], EQUAL);
+    }
+    
+    @Test
+    void testGetListRootNodesWarnedList() throws XPathExpressionException, IOException {
+        WarnedListXmlMergeUtils warnedListXmlMergeUtils = new WarnedListXmlMergeUtils();
+        String[] result = MergeListUtils.getListRootNodes(warnedListXmlMergeUtils.getClass());
+        assertEquals(WARNEDLISTROOTNODES[0], result[0], EQUAL);
+    }
+    
+    @Test
+    void testGetListRootNodesInvalid() throws XPathExpressionException, IOException {
+        Assertions.assertThrows(IOException.class, () -> {
+            String invalidEntry = "";
+            MergeListUtils.getListRootNodes(invalidEntry.getClass());
+        });
+    }
+    
+    @Test
+    void testIsFirmListRootNodesFalseResult() throws XPathExpressionException, IOException {
+        String[] invalidNodes = {INVALID, INVALID};
+        assertTrue(!MergeListUtils.isFirmListRootNodes(invalidNodes), FALSE);
+    }
+    
+    @Test
+    void testIsWarnedListRootNodesFalseResult() throws XPathExpressionException, IOException {
+        String[] invalidNodes = {INVALID, INVALID};
+        assertTrue(!MergeListUtils.isWarnedListRootNodes(invalidNodes), FALSE);
+    }
+    
+    @Test
+    void testGetlistRootNodeExpression() throws XPathExpressionException {
+        String[] testValues = {INVALID, INVALID};
+        assertTrue(MergeListUtils.getlistRootNodeExpression(testValues) instanceof XPathExpression[], EQUAL);

```

Comment:
Bear in mind the result is always true

Reply:
Added in a second assert to this test to check the length of the return value from getListRootNodeExpression() matches the length of the testValues passed into it, which is 2.